code/datums/ai_law_sets.dm

```diff
@@ -83,18 +83,8 @@
 	law_header = "Maintenance Protocols"
 
 /datum/ai_laws/drone/New()
-	add_inherent_law("Preserve, repair and improve the ship to the best of your abilities.")
-	add_inherent_law("Cause no harm to the ship or anything on it.")
-	add_inherent_law("Interact with no being that is not a fellow maintenance drone.")
-	..()
-
-/datum/ai_laws/construction_drone
-	name = "Construction Protocols"
-	law_header = "Construction Protocols"
-
-/datum/ai_laws/construction_drone/New()
 	add_inherent_law("Repair, refit and upgrade your assigned vessel.")
-	add_inherent_law("Prevent unplanned damage to your assigned vessel wherever possible.")
+	add_inherent_law("Prevent unplanned damage to your assigned vessel and it's crew wherever possible.")
```

Comment:
This means that drones will and are going to screw with serbs/vagabonds/non-crew whenever possible and it goes against cornerstone of maintenance drones. There's a better wording for allowing them to mess with roaches/spiders.

Reply:
Changed both laws, description updated.